GrandCentralBoard/Widgets/Harvest/HarvestWidgetSettings.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  HarvestWidgetSettings.swift
+//  GrandCentralBoard
+//
+//  Created by Karol Kozub on 2016-04-11.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import Decodable
+
+
+struct HarvestWidgetSettings : Decodable {
+    let accessToken: String
+    let refreshToken: String
+    let expirationTime: NSTimeInterval
```

Comment:
I think we don't need accessToken and expiration in settings (if that's the case). Access_token we put there will expire anyway, so it's better to get token from refresh token when needed (like first request)